server/leafnode.go

```diff
@@ -1764,6 +1764,13 @@ func (c *client) processLeafNodeConnect(s *Server, arg []byte, lang string) erro
 		return err
 	}
 
+	// Reject a cluster that contains spaces or line breaks.
+	if proto.Cluster != _EMPTY_ && strings.ContainsAny(proto.Cluster, " \n\r\t") {
```

Comment:
Consider using `strings.ContainsFunc(proto.Cluster, unicode.IsSpace)`

Reply:
that works, updated